templates/tab/js/default/public/auth-start.html

```diff
@@ -12,6 +12,7 @@
       integrity="sha384-mhp2E+BLMiZLe7rDIzj19WjgXJeI32NkPvrvvZBrMi5IvWup/1NUfS5xuYN5S3VT"
       crossorigin="anonymous"
     ></script>
+    <script type="text/javascript" src="https://alcdn.msauth.net/browser/2.21.0/js/msal-browser.min.js"></script>
```

Comment:
same with another comment, let's see whether integrity and crossorigin should be added

Reply:
Fixed. Thx!